ScanUnsupportedChars/ScanUnsupportedChars.ps1

```diff
@@ -72,13 +80,19 @@
       .\ScanUnsupportedChars.ps1  -SharePath  \\server\SyncShare  -CsvPath "C:\temp"
       This would scan unsupported file names for the provided remote SyncShare and puts the output in the
       CSV formatted files in the output directory.
+ 
+ .EXAMPLE
+      .\ScanUnsupportedChars.ps1  -SharePath  \\server\SyncShare  -DestinationPath "\\some\server" -RoboCopyOptions '/SEC'
+      This would scan unsupported file names for the provided remote SyncShare and puts the output in the
+      CSV formatted files in the output directory.
```

Comment:
Output directory is not specified here.

Reply:
I've updated the comment as the last line was wrong, for the copy directory a new parameter (DestinationPath) is beeing used and mentioned in the example to not conflict if users might want to have csv report of the affected files additionally